gems/sorbet-runtime/lib/types/compatibility_patches.rb_27.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
Can we make sure to either add something in the file path and/or add a preamble to the file indicating which files are autogenerated and should not be edited manually?

Reply:
I think adding something in the file path is easier than trying to shoehorn it in after the sigil lines.  I will make them `$f_27_autogenerated.rb` instead.

Reply:
All of our other autogenerated files have preambles (e.g., every autogenerated RBI file).
We do this so we can show the command you need to run to regenerate it.

Reply:
OK, then I guess that wasn't an "or" ask. :)

Reply:
i guess in my mind the `or` was for the path name change (i don't mind if it mentions autogenerated in the path or not)😅